scripts/veloxbench/veloxbench/cpp_micro_benchmarks.py

```diff
@@ -0,0 +1,121 @@
+import copy
+import json
+import os
+import tempfile
+
+import conbench.runner
+
+from veloxbench import _benchmark
+
+RUN_OPTIONS = {
+    "iterations": {
+        "default": None,
+        "type": int,
+        "help": "Number of iterations of each benchmark.",
+    },
+}
+
+
+# TODO: what other COMMON_OPTIONS?
+# https://github.com/ursacomputing/benchmarks/blob/033eee0951adbf41931a2de95caccbac887da6ff/benchmarks/cpp_micro_benchmarks.py#L18-L59
+# TODO: the suite filter won't quite work yet either
+COMMON_OPTIONS = {
+    "suite-filter": {
+        "default": None,
+        "type": str,
+        "help": "Regex filtering benchmark suites.",
+    },
+}
+
+
+def get_run_command(filename, options):
+    # TODO: we'll probably want to unify these runner scripts
+    # Must run `make benchmarks-basic-build` before this
+    # TODO: deal with the paths better (especially to benchmarks/basic)
+    command = [
+        "../../../scripts/benchmark-runner.py",
+        "run",
+        "--path",
+        "../../../_build/release/velox/benchmarks/basic/",
+        "--dump-path",
+        filename,
+    ]
+
+    # TODO: extend cpp micro benchmarks to allow for iterations
+    iterations = options.get("iterations", None)
+    if iterations:
+        raise NotImplementedError()
+
+    return command
+
+
+def _parse_benchmark_name(full_name):
+    # TODO: Do we need something more complicated?
+    # https://github.com/ursacomputing/benchmarks/blob/033eee0951adbf41931a2de95caccbac887da6ff/benchmarks/cpp_micro_benchmarks.py#L86-L103
+    return {"name": full_name}
```

Comment:
In Arrow, the names of the microbenchmarks contains useful info that gets separated out, I don't see that in velox (yet?)

Reply:
Yes, for now we only use the binary name, which describes what the benchmark is supposed to exercise. What kind of information is baked into the benchmark name in Arrow?

Reply:
In Arrow (though I think this is actually consequence of using googlebench), the parameters specified to benchmark across like at https://github.com/apache/arrow/blob/fc06daed1d399959cb0fcab458e58f7d7cabc7c1/cpp/src/arrow/compute/kernels/vector_sort_benchmark.cc#L286-L293 will be added to the name of the benchmark, like: `TableSortIndicesInt64Narrow/1048576/100/16/32`, `TableSortIndicesInt64Narrow/1048576/100/16/4`, `TableSortIndicesInt64Narrow/1048576/100/16/1`. And then in our processing step we split those up so that the name bit at the beginning and the parameters at the end are separated (though both recorded so we know that we're comparing one benchmark with the same set of parameters when we are making comparisons, of course!)

And you can see how this results in the Arrow conbench at:
https://conbench.ursa.dev/runs/96adab6a408649adbd722c9758d4d895/?search=TableSortIndicesInt64Narrow

Reply:
There might not be any in Folly if this isn't how expanding across a number of case values works there